docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -1,25 +1,38 @@
 # SSPCommonTemplatesModificationReverted
+<!--apinnick Nov 2022-->
 
 ## Meaning
 
-This alert indicates that one or more common templates were changed by the ssp operator as part of its reconciliation procedure.
-If any changes were made to a common template manually or by a script they were reverted by the operator.
+This alert fires when the Scheduling, Scale, and Performance (SSP) Operator reverts changes to common templates as part of its reconciliation procedure.
+
+The SSP Operator is responsible for deploying and reconciling the common templates and the Template Validator. If a user or script changes a common template, the changes are reverted by the SSP Operator.
 
 ## Impact
 
-The changes made manually on the templates are lost.
-## Diagnosis
+Changes to common templates are overwritten.
 
-The common templates should never be modified manually, find out if you are misusing the common templates by making such modifications.
+## Diagnosis
 
-- Check the logs of the SSP Operator pods:
-	- `export NAMESPACE="$(kubectl get deployment -A | grep ssp-operator | awk '{print $1}')"`
-	- `kubectl -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator`
-	- Look for lines similar to this to find the names of the restored Common Templates:
-	  `Changes reverted in common template: <template name>`
+1. Export the `NAMESPACE` environment variable:
+  ```bash
+  $ export NAMESPACE="$(kubectl get deployment -A | grep ssp-operator | awk '{print $1}')"
+  ```
+2. Check the `ssp-operator` logs for templates with reverted changes:
+  ```bash
+  $ kubectl -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator
```

Comment:
@sradco 
Would it make sense to add a `| grep 'Changes reverted'` to this command?

Reply:
@borod108 please see Avital's question

Reply:
@borod108 ping

Reply:
yes, good idea!

Reply:
maybe we can add "-C 3" to the command tough to see context.